tests/E2E/test/selectors/BO/international/translations.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     translations_type:'//*[@id="form_modify_translations_translation_type"]',
     translations_language:'//*[@id="form_modify_translations_language"]',
     modify_button:'//*[@id="main-div"]//form[@name="modify_translations"]//button[@class="btn btn-primary"]/span',
-    shop_button:'(//div[@label="Shop"])[31]',
```

Comment:
it's a super risky and error prone selector: can't we do better? :(

Reply:
Thank you for your review! 
Yes it's risky, I changed it by =>  //*[@id="app"]//li[2]/div[@label="Shop"]

Reply:
@mickaelandrieu do you accept the change request?